web/ui/module/codemirror-promql/README.md

```diff
@@ -1,13 +1,18 @@
 CodeMirror-promql
 =================
-[![CircleCI](https://circleci.com/gh/prometheus-community/codemirror-promql.svg?style=shield)](https://circleci.com/gh/prometheus-community/codemirror-promql) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE)
+[![CircleCI](https://circleci.com/gh/prometheus-community/codemirror-promql.svg?style=shield)](https://circleci.com/gh/prometheus-community/codemirror-promql) [![GitHub license](https://img.shields.io/badge/license-Apache-blue.svg)](./LICENSE)
 [![NPM version](https://img.shields.io/npm/v/codemirror-promql.svg)](https://www.npmjs.org/package/codemirror-promql) [![codecov](https://codecov.io/gh/prometheus-community/codemirror-promql/branch/master/graph/badge.svg?token=1OSVPBDKZC)](https://codecov.io/gh/prometheus-community/codemirror-promql)
 
 ## Overview
 
 This project provides a mode for [CodeMirror Next](https://codemirror.net/6) that handles syntax highlighting, linting
 and autocompletion for PromQL ([Prometheus Query Language](https://prometheus.io/docs/introduction/overview/)).
 
+## Where does it come from?
+
+This repo is synced from https://github.com/prometheus/prometheus/tree/main/web/ui/module/codemirror-promql. 
```

Comment:
It's slightly confusing because this README.md is also in the `prometheus/prometheus` repo. How about "The authoritative copy of this code lives in [...] and is synced to [...] on a regular basis by a bot. Please contribute any code changes to the code in [...]."